Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Use the official Ubuntu 24.04 image as the base image
+FROM ubuntu:24.04
+
+# Set the working directory
+WORKDIR /usr/src/app
+
+# Install necessary dependencies
+RUN apt-get update && apt-get install -y \
+    curl \
+    protobuf-compiler \
+    xz-utils
```

Comment:
If we use the nix this would be brought in there

Reply:
fixed